src/components/Article.jsx

```diff
@@ -0,0 +1,24 @@
+import { useParams } from "react-router-dom";
+import { getArticle } from "../api";
+import { useEffect, useState } from "react";
+
+export const Article = () => {
+  const [article, setArticle] = useState({});
+  const { article_id, title } = useParams();
+
+  useEffect(() => {
+    getArticle(article_id, title).then(({ article }) => {
+      setArticle(article);
+    });
+  }, []);
+
+  return (
+    <section>
+      <h2>{article.title}</h2>
+      <p>{article.author}</p>
+      <p>{article.created_at}</p>
+      <p>Topic: {article.topic}</p>
+      <p>{article.body}</p>
+    </section>
+  );
```

Comment:
This is great! I definitely think you've made enough progress to start to consider adding in loading state, and possibly even error state to your page for the API call